### DISCUSSION TITLE
pb_hooks not working with building pocketbase go

### DISCUSSION THREAD

**Discussion by starredev** [pocketbase/pocketbase#4477] [Q&A]
pb_hooks does not work on my own generated executable of pocketbase.
I've followed https://pocketbase.io/docs/go-overview/

Am I doing anything wrong?

**Comment by ganigeorgiev** [comment]
The JSVM plugin is not enabled by default when using Go. This is not documented at the moment but you could check the prebuilt [main.go file](https://github.com/pocketbase/pocketbase/blob/master/examples/base/main.go).

```go
jsvm.MustRegister(app, jsvm.Config{
    HooksWatch:    true,
    HooksPoolSize: 25,
})
```

**Reply by starredev** [reply]
That works! Thanks :)
